client/packages/requisitions/src/RequestRequisition/api/api.ts

```diff
@@ -348,4 +349,29 @@ export const getRequestQueries = (sdk: Sdk, storeId: string) => ({
     const result = await sdk.supplierProgramSettings({ storeId });
     return result.supplierProgramRequisitionSettings;
   },
+  getIndicators: async (
+    customerNameLinkId: string,
+    periodId: string,
+    programId: string
+  ) => {
+    let result = await sdk.programIndicators({
+      storeId,
+      customerNameLinkId,
+      periodId,
+      programId,
+    });
+
+    if (result?.programIndicators.__typename === 'ProgramIndicatorConnector') {
+      return result.programIndicators;
+    }
+  },
+  updateIndicatorValue: async (patch: UpdateIndicatorValueInput) => {
+    let result = await sdk.updateIndicatorValue({ storeId, input: patch });
+
+    if (!!result?.updateIndicatorValue) {
+      return result.updateIndicatorValue;
+    }
+
+    throw new Error('Could not update indicator value');
```

Comment:
this guy isn't handled if it does get called 😱 
I'd expect a toast but I get a full error - the problem is just that the error handler later on is expecting a `__typename` prop which won't exist here. I've added a toast and fix in my suggestion.